cannon/mipsevm/tests/fuzz_evm_common64_test.go

```diff
@@ -0,0 +1,54 @@
+//go:build cannon64
+// +build cannon64
+
+package tests
+
+import (
+	"math/big"
+	"os"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+
+	"github.com/ethereum-optimism/optimism/cannon/mipsevm/arch"
+	"github.com/ethereum-optimism/optimism/cannon/mipsevm/testutil"
+)
+
+func FuzzStateDmultInsn(f *testing.F) {
+	versions := GetMipsVersionTestCases(f)
+	f.Fuzz(func(t *testing.T, rs arch.Word, rt arch.Word, seed int64) {
+		for _, v := range versions {
+			t.Run(v.Name, func(t *testing.T) {
+				goVm := v.VMFactory(nil, os.Stdout, os.Stderr, testutil.CreateLogger(), testutil.WithRandomization(seed), testutil.WithPC(0), testutil.WithNextPC(4))
```

Comment:
just fyi - we also have this `WithPCAndNextPC` util for less noise:
```suggestion
				goVm := v.VMFactory(nil, os.Stdout, os.Stderr, testutil.CreateLogger(), testutil.WithRandomization(seed), testutil.WithPCAndNextPC(0))
```